### DISCUSSION TITLE
What makes Z-score std a little bit better in unit 3.7 Feature Normalization (Parts 1-2).

### DISCUSSION THREAD

**Discussion by jihobak** [Lightning-AI/dl-fundamentals#47] [Lecture videos and quizzes] [answered]
From my understanding, It might be ... there are more chance to get gradient from both sides (+), (-) because of zero centering?

**Comment by rasbt** [comment, marked as answer]
Yes that's right. You can have positive and negative values with the gradient and the weights. So there are 4 different settings negative weight & negative input, negative weight & positive input, positive weight & negative input, positive weight & positive input. If you normalize to [0, 1] range, you can only have negative weight & positive input, and positive weight & positive input.

**Reply by jihobak** [reply]
wow short and clean thanks :)
